synapse/storage/engines/postgres.py

```diff
@@ -15,6 +15,8 @@
 import logging
 from typing import TYPE_CHECKING, Any, Mapping, NoReturn, Optional, Tuple, cast
 
+import psycopg2
```

Comment:
I reverted this change in [c92f2f7](https://github.com/matrix-org/synapse/pull/12974/commits/c92f2f740996e7beb3887476b71476c8674198b6) as it was causing errors.